src/StreamsInfo.js

```diff
@@ -21,20 +21,24 @@ class StreamsInfo {
             throw new TypeError('You should provide a correct url, bastard.');
         }
 
-        const {ffprobePath, timeoutInSec} = config;
+        const {ffprobePath, timeoutInMs} = config;
 
         if (!_.isString(ffprobePath) || _.isEmpty(ffprobePath)) {
             throw new Errors.ConfigError('You should provide a correct path to ffprobe, bastard.');
         }
 
-        if (!_.isInteger(timeoutInSec) || timeoutInSec <= 0) {
+        if (!_.isInteger(timeoutInMs) || timeoutInMs <= 0) {
             throw new Errors.ConfigError('You should provide a correct timeout, bastard.');
         }
 
         this._assertExecutable(ffprobePath);
 
-        this._config = config;
-        this._url    = url;
+        this._config = {
+            ffprobePath,
+            timeout: timeoutInMs * 1000
```

Comment:
It looks like an error.

Reply:
FFprobe expects a value in microseconds. That is why I make a one-time conversion.